check-netbox-machine-state.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+from functools import reduce
+from subprocess import DEVNULL, CalledProcessError, run
+
+import pynetbox
+
+ping_cmd = "/usr/bin/ping"
```

Comment:
It is probably not a big deal but I'm wondering whether we should hard-code the absolute path.

Reply:
Which leaves the decision to something else. Also it only fixes ruff-warnings because it doesn't know what it does. I changed it now.